standard/docs/en/schema/merging.md

```diff
@@ -1,30 +1,31 @@
-# Merging 
+# Merging
 
-In OCDS, merging involves combining individual [releases](../getting_started/releases_and_records.md) of data during a contracting process into a [record](../getting_started/releases_and_records.md) which provides an overview of the current state of that process. Including a versioned release as part of the merged record provides structured information on the history of the process: showing when updates were made, or values were changed. 
+An OCDS [record](../getting_started/releases_and_records.md) aggregates all the releases available for a contracting process at a given time, and optionally includes:
 
-<div class="example hint" markdown=1>
+* a compiled release, which expresses the current state of the contracting process, by showing only the most recent field values
+* a versioned release, which expresses all historical states of the contracting process, by showing all the field values over time
 
-<p class="first admonition-title">Worked Example</p>
+**Merging** is the process of combining individual releases into a compiled or versioned release, described in more detail below.
 
-A publisher provides a tender release on 1st January, with a planned contract value of $1000.
+The versioned release makes it easy to see how field values have changed over time: for example, how contract values have been modified or how milestones have been re-scheduled during implementation.
```

Comment:
This is a bit jarring in context of the previous paragraphs. We go from "Records are this." to "Records make use of merging" to "Merging is used for compiled and versioned releases and here are details" to "Here's a repeat of the versioned release details".

Recommend merging this content with line 6 to avoid duplication and creating a better flow for the introduction.

Reply:
Yeah, that last paragraph was on the original page and I didn't know where to put it. The worked example demonstrates the utility of the versioned release, so I deleted this paragraph in 50ee7f9